unliftio/src/UnliftIO/Async.hs

```diff
@@ -42,26 +47,47 @@ module UnliftIO.Async
     mapConcurrently, forConcurrently,
     mapConcurrently_, forConcurrently_,
     replicateConcurrently, replicateConcurrently_,
-    Concurrently(..),
+
+#if MIN_VERSION_base(4,8,0)
+    Concurrently (..),
```

Comment:
Why is this conditionally exposed? It was previously exported without the condition.

Reply:
A typo, good find